src/main/kotlin/no/nav/familie/ef/sak/api/beregning/VedtakService.kt

```diff
@@ -0,0 +1,23 @@
+package no.nav.familie.ef.sak.api.beregning
+
+import no.nav.familie.ef.sak.repository.VedtakRepository
+import no.nav.familie.ef.sak.repository.domain.InntektWrapper
+import no.nav.familie.ef.sak.repository.domain.PeriodeWrapper
+import no.nav.familie.ef.sak.repository.domain.Vedtak
+import org.springframework.stereotype.Service
+import java.math.BigDecimal
+import java.math.RoundingMode
+import java.util.*
+
+@Service
+class VedtakService(private val vedtakRepository: VedtakRepository) {
+
+    fun lagreVedtak(vedtakRequest: VedtakRequest, behandlingId: UUID): UUID {
+        return vedtakRepository.insert(Vedtak(behandlingId,
```

Comment:
Her må vi "bumpe" steg på behandlingen, men det tar vi i egen oppgave.

Reply:
Burde vi "slette" gammelt vedtak hvis man utfører denne to ganger? 

Eks: 
1 Lag vedtak
2 Angre seg, endre et vilkår og lag vedtak på nytt
3 Da ønsker vi kanskje ikke å ha to vedtak i systemet? 🤔